test/integration/client-side-encryption/client_side_encryption.prose.test.js

```diff
@@ -1106,6 +1110,77 @@ describe('Client Side Encryption Prose Tests', metadata, function () {
 
     it.skip('Via bypassAutoEncryption', () => {}).skipReason =
       'TODO(NODE-2422): Implement "Bypass spawning mongocryptd" tests';
+
+    describe('via loading shared library', function () {
+      let clientEncrypted;
+      let client;
+      // Setup
+      beforeEach(async function () {
+        const { cryptSharedLibPath } = getEncryptExtraOptions();
+        if (!cryptSharedLibPath) {
+          this.skip('Skipped because CRYPT_SHARED_LIB_PATH is missing');
+        }
```

Comment:
```suggestion
        if (!cryptSharedLibPath) {
          this.currentTest.skipReason = 'Skipped because CRYPT_SHARED_LIB_PATH is missing';
          this.skip();
        }
```

To attach skip reasons, we assign the reason to the `skipReason` on either `currentTest` or `test` (depending on the place where the test is being skipped from).